kodilla-hibernate/src/main/java/com/kodilla/hibernate/invoice/Invoice.java

```diff
@@ -0,0 +1,73 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table (name = "INVOICES")
+public class Invoice {
+
+    private int id;
+    private String number;
+    private List<Item> items;
+
+    public Invoice(String number) {
+        this.number = number;
+        this.items = new ArrayList<>();
+    }
+
+    public Invoice(){}
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name= "ID", unique = true)
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    @Column(name = "INVOICE_NUMBER")
```

Comment:
proponuje nazywac kolumny jak najprosciej, jak mamy cos w tabeli invoices, to kolumna nie musi sie nazywac invoice_costam, a wystarczy costam (podobnie jak przy nazywaniu pol w klasach)